resources/sushy-tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.hub.docker.com/library/python:3.9
+FROM "${OVERRIDE_DOCKER_REGISTRY_2}/"library/python:3.9
```

Comment:
This is not needed, we are not building it in dev-env.

Reply:
done